icicle-compiler/src/Icicle/Sea/IO/Zebra.hs

```diff
@@ -174,7 +174,6 @@ seaOfDefReadProgram state = vsep
   , "        /* run compute on the facts read so far */"
   , "        if (*fact_count != 0) {"
   , indent 12 $ vsep $ fmap (\i -> pretty (nameOfCompute i) <+> " (program);") computes
-  , "            *fact_count = 0;"
```

Comment:
Curious why no longer gets reset to 0? I introduced that [here](https://github.com/ambiata/icicle/pull/535/files) to fix a bug is all.